website/api/controllers/deliver-apple-csr.js

```diff
@@ -139,7 +139,7 @@ module.exports = {
       template: 'email-signed-csr-for-apns',
       templateData: {},
       attachments: [{
-        contentBytes: generateCertificateResult.request,
+        contentBytes: Buffer.from(generateCertificateResult.request).toString('base64'),
```

Comment:
I'm not a 100% sure if this is the Sails way to accomplish this.

Reply:
Yes, that works!